tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -205,7 +215,7 @@ func (a *awsProvider) getNetworkInterface(infraID string) (*ec2.InstanceNetworkI
 // createInstance creates one VM instance based on the given information and returns a instance struct with all its
```

Comment:
Please add information about the parameters of the function. With userData specify that it is plaintext.

Reply:
make sure you include the function of userData, that it will be executed on instance startup

Reply:
Got it.